test/python/test_nixl_api.py

```
@@ -22,11 +22,91 @@
+    nixl_agent,
+    nixl_agent_config,
+    nixl_thread_sync_t,
+    nixl_xfer_handle,
+)
 
 # NIXL pytest fixtures
 
 
+def test_xfer_handle_cancel():
+    class AgentStub:
+        def __init__(self):
```

Ruff (0.16.3)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)